pkg/typesystem/typesystem.go

```diff
@@ -174,6 +175,7 @@ type TypeSystem struct {
 	modelID                 string
 	schemaVersion           string
 	authorizationModelGraph *graph.AuthorizationModelGraph
+	authzWeightedGraph      *graph.WeightedAuthorizationModelGraph
```

Comment:
Noting this here just for others, feel free to ignore. With the current implementation of `WeightedAuthorizationModelGraph` you can't remove the `authModelGraph` and just keep weighted one, because the methods in authModelGraph are not exposed through the weighted one.

Reply:
Track via https://github.com/openfga/openfga/issues/2119